pandas/core/window.py

```diff
@@ -749,12 +749,13 @@ def _apply_window(self, mean=True, **kwargs):
             additional_nans = np.array([np.NaN] * offset)
 
             def f(arg, *args, **kwargs):
+                cfunc = getattr(libwindow, func)
```

Comment:
you you move the _apply from Rolling here and use that? (I don't remember fully why i didn't do that iniitally); but that is more flexible and really *should* be in the base class